lib/extensions/src/lib/services/extension.service.ts

```diff
@@ -272,12 +265,12 @@ export class ExtensionService {
      * @param context Parameter object for the expression with details of app state
      * @returns Result of evaluated expression, if found, or the literal value otherwise
      */
-    runExpression(value: string | any , context?: any) {
-        if (typeof value === 'string' ) {
+    runExpression(value: string | any, context?: any) {
```

Comment:
could you please add return type?

Reply:
seems it should be 
` runExpression(value: string | any, context?: any): string | any {}`